crates/network-scanner-net/examples/intense_tcp.rs

```diff
@@ -0,0 +1,107 @@
+use std::{
+    mem::MaybeUninit,
+    net::{SocketAddr, SocketAddrV4},
+};
+
+use network_scanner_net::socket::AsyncRawSocket;
+use socket2::SockAddr;
+use tokio::task::JoinHandle;
+
+/// This example needs to be run with a echo server running on a different process
+/// ```
```

Comment:
suggestion: It would be nice if it was possible to choose between server / client using a CLI argument instead of modifying and recompiling the code.

Reply:
added